### PR TITLE
Sage 133: added builds via Jenkins

### PR DESCRIPTION
**Pull request by wgerlach**
the testing harness (for the new Jenkins) works locally but not with github actions

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -16,9 +16,15 @@ jobs:
...
+        export USE_HOST_DOCKER=1
+        
+        ./run.sh -d
+
+    - name: wait for environment
+      run: while [ $(curl -s localhost:5000/healthy)_ != "ok_"  ] ; do echo waiting... ; docker-compose logs --tail=20 ; echo "jenkins log:" ; docker logs jenkins --tail=20 ; sleep 2 ; done
 
     - name: run tests and coverage
-      run: docker-compose run --rm  sage-ecr /bin/ash -c 'coverage run -m pytest -v &&  coverage report -m'
+      run: docker exec sage-ecr_sage-ecr_1 /bin/ash -c 'coverage run -m pytest -v &&  coverage report -m  --fail-under 70'
```
Comment (jswantek) [inline .github/workflows/main.yml:29]: You could have just changed the `.coveragerc` file from `fail_under=85` to `fail_under=70`

**Comment by jswantek** [comment]
I noticed the following and wanted to bring them to your attention

In the 'wait for environment' step
`/home/runner/work/_temp/d675ac47-e318-4c1d-a990-0cd4f9ee0408.sh: line 1: [: too many arguments`

and then in the 'run tests and coverage' steps
`test_ecr_api.py::test_upload_and_build SKIPPED `
^ did you mean for this test to be "skipped"?

**Comment by jswantek** [comment]
Ah.  I see now that you specifically skipped the "slow" tests.  which is why you said: "the testing harness (for the new Jenkins) works locally but not with github actions"

File: .github/workflows/main.yml (hunk 1 above)
Comment (wgerlach) [reply .github/workflows/main.yml:29]: As you noticed yourself below, this is a specific workaround for github actions, thus I did not want to modify the default behavior.

**Comment by wgerlach** [comment]
@jswantek  Regarding the `too many arguments` error message, yes I saw that, but I am not sure where that comes from, but the 'wait for environment' step seems to be working fine despite that.

**Pull request merged**